inference-engine/src/cldnn_engine/cldnn_engine.cpp

```diff
@@ -208,9 +222,21 @@ InferenceEngine::ICNNNetwork::Ptr clDNNEngine::CloneAndTransformNetwork(const In
                         }
                         return true;
                     });
+
+            if (enableInt8) {
+                pass_config->set_callback<ngraph::pass::ConvertQuantizeDequantize>([](const_node_ptr &node) -> bool {
+                    const std::shared_ptr<ngraph::Node> multiply = const_cast<ngraph::Node*>(node.get())->shared_from_this();
```

Comment:
```
auto multiply = std::const_pointer_cast<ngraph::Node>(node);
```

And why do you even need to have it non-constant in transformation callback?

Reply:
Done, fixed:  was removed for `areQuantizeAndDequantizeSupportedForSubtract`, but was moved into `areQuantizeAndDequantizeSupportedForMultiply`. 
The type cast reason in `areQuantizeAndDequantizeSupportedForMultiply` method is nGraph LPT utils (similar as nGraph core utils: core\include\ngraph\graph_util.hpp) interfaces: `std::shared_ptr<ngraph::Node>` is used as input and output arguments.